pkg/mock/runner.go

```diff
@@ -193,6 +205,19 @@ func (m *ShellMockRunner) Stderr(err io.Writer) {
 	m.stderr = err
 }
 
+// Cleanup for ExecMockRunner
+func (m *ShellMockRunner) Cleanup() {
+	m.Dir = ""
```

Comment:
why string here but not nil?

Reply:
not allowed in go. strings are initialised as "".